ts/state/ducks/SessionTheme.tsx

```diff
@@ -334,10 +334,16 @@ export const SessionGlobalStyles = createGlobalStyle`
     --margins-md:  15px;
     --margins-lg:  20px;
 
+    /* PADDING */
+    --padding-message-content: 7px 13px;
```

Comment:
This could be used in the FIXME lines above?

Reply:
yes that's the one